cumulus/pallets/collator-selection/src/lib.rs

```diff
@@ -170,10 +174,14 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxInvulnerables: Get<u32>;
 
-		// Will be kicked if block is not produced in threshold.
+		/// Will be kicked if block is not produced in threshold.
 		#[pallet::constant]
 		type KickThreshold: Get<BlockNumberFor<Self>>;
 
+		/// The additional period of time the candidate deposits are reserved for after being kicked
+		/// or leaving the candidate set.
+		type UnbondingPeriod: Get<BlockNumberFor<Self>>;
```

Comment:
I think we should be more precise on what are the consequences of this value.

From what I understand kicking somebody from the candidate list now cost the lock of their bid for `UnbondingPeriod`.

Also I don't exactly understand the overall selection, the bid is not locked for the duration of the session when the collator is selected, and the moment of `new_session` is quite deterministic in polkadot, so the collator is chosen on how much they can reserve at the session boundary for the duration `UnbondingPeriod`.